tests/e2e/fixtures/cairo_0_1_mixed/protostar.toml

```diff
@@ -1,7 +1,6 @@
 [project]
 protostar-version = "0.9.1"
 lib-path = "lib"
-linked-libraries = ["src"]
```

Comment:
Why was this deleted?

Reply:
`linked-libraries` paths should lead to folders with module declaration file (`lib.cairo`). There is no such file in `src`

Reply:
https://github.com/software-mansion/protostar/actions/runs/4513087426/jobs/7947598500#step:8:383